snapshot-labs/sx-monorepo
# chore: update oz votes strategy address

### Summary

With this change OZ votes trace 208 is updated to match Herodotus contracts (using u32 for some tree properties).

Closes: https://github.com/snapshot-labs/sx-monorepo/issues/997 (via Checkpoint update).

### How to test

**Currently broken because Infura Starknet mainnet API is timing out.**

1. Edit your space to use this **new** contract.
2. Create proposal.
3. After a while you see your VP.
